apps/src/lib/levelbuilder/lesson-editor/ProgrammingExpressionsEditor.jsx

```diff
@@ -0,0 +1,196 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {
+  programmingExpressionShape,
+  programmingEnvironmentShape
+} from '@cdo/apps/lib/levelbuilder/shapes';
+import color from '@cdo/apps/util/color';
+import Dialog from '@cdo/apps/templates/Dialog';
+import {connect} from 'react-redux';
+import {removeProgrammingExpression} from '@cdo/apps/lib/levelbuilder/lesson-editor/programmingExpressionsEditorRedux';
+import * as Table from 'reactabular-table';
+import {lessonEditorTableStyles} from './TableConstants';
+
+const styles = {
+  search: {
+    paddingBottom: 10
+  },
+  actionsColumn: {
+    display: 'flex',
+    justifyContent: 'space-evenly',
+    backgroundColor: 'white'
+  },
+  remove: {
+    fontSize: 14,
+    color: 'white',
+    background: color.dark_red,
+    cursor: 'pointer',
+    textAlign: 'center',
+    width: '100%',
+    lineHeight: '30px'
+  },
+  dropdown: {
+    width: '100%'
+  }
+};
+
+class ProgrammingExpressionsEditor extends Component {
+  static propTypes = {
+    programmingEnvironments: PropTypes.arrayOf(programmingEnvironmentShape)
```

Comment:
This doesn't seem needed for this PR but I assume it will be for search and create? (If so, totally fine to leave it in!)

Reply:
Ya I pulled apart another PR I had going in order to get this in.